master/about.md~ba18ad8... add v0.9.0 docs to site

```diff
@@ -0,0 +1,71 @@
+# How Ark Works
```

Comment:
This looks like a garbage file (git leftover from something or other), should be removed